### DISCUSSION TITLE
How can i restart or reload a poller's Centreon Engine

### DISCUSSION THREAD

**Discussion by hqm199** [centreon-archive/centreon-archived#9807] [Q&A] [answered]
hi  I install centreon 21.04.0  and config a poller with it . when i export configuration to a poller , like this
Generate Configuration Files
Run monitoring engine debug (-v)
Move Export Files
Restart Monitoring Engine | Method  	Restart 	Reload

It doesn't  woke  which one you choose  Restart    or Reload  ,I can confirm cfg files has changed after this ,but why poller's Centreon Engine can't  Restart  or Reload ?

but if i restart or reload Centreon Engine Manual poller ,it works right  , It can get service check result  in centreon web.

English is not my native language, so I made some mistakes, but I hope you can understand my problem

**Comment by lpinsivy** [comment, marked as answer]
Hi, did you check in /var/log/centreon-gorgone/gorgoned.log log file?

Maybe centreon-gorgone user/group is not part of centreon-engine group

**Reply by ponchoh** [reply]
https://www.centreon.com/en/blog/centreon-troubleshoot-series-episode-1-help-my-poller-is-down/
